vulkaninfo/vulkaninfo.c

```diff
@@ -5561,7 +5560,11 @@ int main(int argc, char **argv) {
     AppCreateInstance(&inst);
 
     if (html_output) {
+#ifdef _WIN32
+        fopen_s(&out, "vulkaninfo.html", "w");
```

Comment:
Can't tell from MSDN docs if failure will assign out=nullptr, the docs say to check the return value.  So perhaps something like
```
if(fopen_s(&out, "vulkaninfo.html", "w") !=0) out=nullptr;
```